tests/phpunit/api/v3/MembershipTest.php

```diff
@@ -147,44 +147,6 @@ public function testMembershipDeletePreserveContribution() {
     $this->assertDBRowNotExist('CRM_Contribute_DAO_Contribution', $ContributionCreate['values'][0]['id']);
   }
 
-  /**
-   * Test Activity creation on cancellation of membership contribution.
-   *
-   * @throws \CRM_Core_Exception
-   */
-  public function testActivityForCancelledContribution(): void {
-    // @todo figure out why financial validation fails with this test.
-    $this->isValidateFinancialsOnPostAssert = FALSE;
-    $contactId = $this->ids['Contact']['order'] = $this->createLoggedInUser();
-
-    $this->createContributionAndMembershipOrder();
-    $membershipID = $this->callAPISuccessGetValue('MembershipPayment', ['return' => 'id']);
-
-    $_REQUEST['id'] = $this->ids['Contribution'][0];
-    $_REQUEST['action'] = 'update';
-    // change pending contribution to completed
-    /* @var CRM_Contribute_Form_Contribution $form */
-    $form = $this->getFormObject('CRM_Contribute_Form_Contribution', [
-      'total_amount' => 100,
-      'financial_type_id' => 1,
-      'contact_id' => $contactId,
-      'payment_instrument_id' => CRM_Core_PseudoConstant::getKey('CRM_Contribute_BAO_Contribution', 'payment_instrument_id', 'Check'),
-      'contribution_status_id' => 3,
-    ]);
-    $form->buildForm();
-    $form->postProcess();
-
-    $this->callAPISuccessGetSingle('Activity', [
```

Comment:
Likewise this seemed to be the only part of this test that was not being checked by the same extension unit test @eileenmcnaughton